compiler/main/docsDriver.cpp

```diff
@@ -17,10 +17,12 @@
  * limitations under the License.
  */
 
+#include "commonFlags.h"
+#include "driver.h"
 #include "docsDriver.h"
```

Comment:
I didn't know about this convention, but it makes sense. I updated this instance and the one in docs.cpp. Thanks!